src/service_layer/utils.py

```diff
@@ -98,25 +98,6 @@ def fetch_missing_entities(self) -> List[models.Party]:
         return data_list
 
 
-def get_repository(
```

Comment:
You should have added the politician's factory creation function to the map.

But I removed this function(get_repository) and moved it to the SqlAlchemyFactory class (=make it a method).

This refactoring enforces clearer modularity.
Pleas ensure that youll add entity and function pair to the map in the SqlAlchemyFactory.